routes/user.js

```diff
@@ -77,6 +78,28 @@ router.route('/review/:netid').get(auth, async(req, res) =>{
   }
 })
 
+router.route('/review/:id').delete(auth, async(req, res) =>{
+  try{
+    let deletedReview = await Review.findByIdAndDelete(req.params.id);
+    let targetUser = await User.findById(deletedReview.target);
+    const index = targetUser.reviews.indexOf(deletedReview);
+    targetUser.reviews.splice(index,1)
+    
```

Comment:
So everything up to this point looks good. I believe that you need to do targetUser.save() right here, so that it will save the fact that you just modified and deleted an element in its review array. So right now the update to the reviews array will not save. However, I did not test it so I could be wrong :)